datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -375,10 +375,9 @@ public void onBecomeLeader() {
     // when an instance becomes a leader, make sure we don't miss new datastreams and
     // new assignment tasks that was not finished by the previous leader
     _eventQueue.put(CoordinatorEvent.createHandleDatastreamAddOrDeleteEvent());
-    _eventQueue.put(CoordinatorEvent.createLeaderDoAssignmentEvent());
-    // This should be only called after createLeaderDoAssignmentEvent as it will verify/cleanup the orphan task nodes
-    // under connector.
-    _eventQueue.put(CoordinatorEvent.createLeaderDoCleanupPostElectionEvent());
+    // verify/cleanup the orphan task nodes under connector should be called only once after becoming leader.
```

Comment:
May just be a good idea to call out why this should only be called once -> that it's an expensive operation.
Someone reading this code later may not realize why without making it explicit.